src/utils/node.ts

```diff
@@ -47,10 +46,10 @@ export const isLiskCoreV3Running = async (liskCorePath: string): Promise<boolean
 	}
 };
 
-const backupDefaultDirectoryIfExists = async (_this: Command) => {
-	if (existsSync(DEFAULT_LISK_DATA_DIR)) {
-		_this.log(`Backing Lisk Core v3 data directory at ${DEFAULT_LISK_DATA_DIR}`);
-		renameSync(DEFAULT_LISK_DATA_DIR, LISK_V3_BACKUP_DATA_DIR);
+const backupDefaultDirectoryIfExists = async (_this: Command, liskCoreV3DataPath: string) => {
+	if (existsSync(liskCoreV3DataPath)) {
+		_this.log(`Backing Lisk Core v3 data directory at ${liskCoreV3DataPath}`);
+		renameSync(liskCoreV3DataPath, LISK_V3_BACKUP_DATA_DIR);
```

Comment:
If `liskCoreV3DataPath` isn't the `DEFAULT_LISK_DATA_DIR`, can we please create the `${homedir()}/.lisk` direcoty and then move it under that?